apps/tests/msg/rpmsg-ping.c

```diff
@@ -134,6 +125,14 @@ int app (struct rpmsg_device *rdev, void *priv)
 
 	LPRINTF("RPMSG endpoint is binded with remote.\r\n");
 	for (i = 0, size = PAYLOAD_MIN_SIZE; i < num_payloads; i++, size++) {
+		struct _payload *i_payload;
+
```

Comment:
i would prefer that you duplicate the exemple, instead of modifying it.
Create a rpmsg_ping_nc.c for the master and a rpmsg_echo_nc.c  in application for the salve

Reply:
OK.